README.md

```diff
@@ -1,6 +1,85 @@
 # Account API
 
-<!-- todo: Write the README -->
+Provides sign in / sign out and attribute storage functionality to
+other GOV.UK applications.
+
+In production this currently uses the [GOV.UK account manager
+prototype][] as an authentication service and the [GOV.UK attribute
+service prototype][] as a user data store.
+
+This app does not serve any user-facing pages.  To see it working you
+will need to run an app which uses it, such as [Finder Frontend][].
+
+After starting Finder Frontend, you should be able to access the
+following links:
+
+- the [Brexit checker journey start page][tc-start]
+- the [Brexit checker results page][tc-results] that reflects the
+  answers you give during the Brexit checker journey
+- the [account sign up page][tc-save-results] to save your answers
+
+When you have set up your local account, you can [sign into your
+account][account-manager].
+
+[GOV.UK account manager prototype]: https://github.com/alphagov/govuk-account-manager-prototype
+[GOV.UK attribute service prototype]: https://github.com/alphagov/govuk-attribute-service-prototype
+[Finder Frontend]: https://github.com/alphagov/finder-frontend
+[account-manager]: http://www.login.service.dev.gov.uk/
+[tc-results]: http://finder-frontend.dev.gov.uk/transition-check/results?c[]=living-ie
+[tc-save-results]: http://finder-frontend.dev.gov.uk/transition-check/save-your-results?c%5B%5D=living-ie
+[tc-start]: http://finder-frontend.dev.gov.uk/transition-check/questions
+
+
```

Comment:
Nitpick, double new line?

Reply:
I normally put a double new line before a h2, though I see I didn't do that for the `Further documentation` heading.